_team_member/eleanor.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: teammember
+team-member-name: Eleanor Stylianides
+team-member-photo: /assets/img/team/Eleanor.jpg
+job-role: Trainee Clinical Informatician
+---
+
+### Bio
+Eleanor joined the CSC team in September 2023 as a Trainee Clinical Scientist in Clinical Informatics to complete the 3-year Scientist Training Programme (STP). Previously she worked as a Business Information Analyst at Northwick Park Hospital, Harrow. She studied Medical Genetics as an undergraduate at Swansea University and has experience in healthtech and in a genetic laboratory.
```

Comment:
For readability its best to separate this into mutliple shorter lines in the markdown. It will not effect how its presented in the html.

Reply:
typo: health tech

Reply:
Updated